docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -168,37 +168,212 @@ Make up your mind on which approach you want to use.
 Want to start a PostgreSQL server on the side with Docker?
 
 [source,shell]
---
+----
 docker run --ulimit memlock=-1:-1 -it --rm=true --memory-swappiness=0 \
            --name postgres-quarkus-hibernate -e POSTGRES_USER=hibernate \
            -e POSTGRES_PASSWORD=hibernate -e POSTGRES_DB=hibernate_db \
            -p 5432:5432 postgres:10.5
---
+----
 
 This will start a non-durable empty database: ideal for a quick experiment!
 ====
 
+[[multiple-persistence-units]]
+=== Multiple persistence units
+
+==== Setting up multiple persistence units
+
+[WARNING]
+====
+This feature is still experimental.
+It is considered stable and usable in production but the configuration properties and CDI integration might evolve depending on your feedback.
+
+If you are using this feature, any feedback (even just to say it fulfills your requirements) is very welcome on our link:{quarkus-mailing-list-index}[quarkus-dev mailing list].
+====
+
+Starting with Quarkus 1.7, you can define multiple persistence units using the Quarkus configuration properties approach.
+
+The properties at the root of the `quarkus.hibernate-orm.` namespace define the default persistence unit.
+For instance, the following snippet defines a default datasource and a default persistence unit:
+
+[source,properties]
+----
+quarkus.datasource.db-kind=h2
+quarkus.datasource.jdbc.url=jdbc:h2:mem:default;DB_CLOSE_DELAY=-1
+
+quarkus.hibernate-orm.dialect=org.hibernate.dialect.H2Dialect
+quarkus.hibernate-orm.database.generation=drop-and-create
+----
+
+Using a map based approach, it is possible to define named persistence units:
+
+[source,properties]
+----
+quarkus.datasource."users".db-kind=h2 <1>
+quarkus.datasource."users".jdbc.url=jdbc:h2:mem:users;DB_CLOSE_DELAY=-1
+
+quarkus.datasource."inventory".db-kind=h2 <2>
+quarkus.datasource."inventory".jdbc.url=jdbc:h2:mem:inventory;DB_CLOSE_DELAY=-1
+
+quarkus.hibernate-orm."users".database.generation=drop-and-create <3>
+quarkus.hibernate-orm."users".datasource=users <4>
+quarkus.hibernate-orm."users".packages=org.acme.model.user <5>
+
+quarkus.hibernate-orm."inventory".database.generation=drop-and-create <6>
+quarkus.hibernate-orm."inventory".datasource=inventory
+quarkus.hibernate-orm."inventory".packages=org.acme.model.inventory
+----
+<1> Define a datasource named `users`.
+<2> Define a datasource named `inventory`.
+<3> Define a persistence unit called `users`.
+<4> Define the datasource used by the persistence unit.
+<5> This configuration property is important but we will discuss it a bit later.
```

Comment:
Do you think we can add a link to the appropriate section?

Reply:
Ah sorry, I should have pinged you too :). 

I don’t want to do it as I prefer people read the doc in order.

Reply:
Makes sense.